godotcord_network_peer.cpp

```diff
@@ -658,6 +659,7 @@ NetworkedMultiplayerGodotcord::NetworkedMultiplayerGodotcord() {
 		discord::User user{};
 		discord::Result result = Godotcord::get_singleton()->get_core()->UserManager().GetCurrentUser(&user);
 		ERR_FAIL_COND(result != discord::Result::Ok);
+        print_line(vformat("Result: %s", static_cast<int>(result)));

```

Comment:
Print int from Result enum

Reply:
You have to print the result before `ERR_FAIL_COND` is executed because `ERR_FAIL_COND` will return from the method, so the print statement is never reached in case of an error